app/src/main/java/nz/org/cacophony/birdmonitor/InternetConnectionFragment.java

```diff
@@ -30,9 +27,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         setUserVisibleHint(false);
 
-        rbNormal = (AppCompatRadioButton) view.findViewById(R.id.rbNormal);
-        rbOnline = (AppCompatRadioButton) view.findViewById(R.id.rbOnline);
-        rbOffline = (AppCompatRadioButton) view.findViewById(R.id.rbOffline);
+        rbNormal = (AppCompatRadioButton) view.findViewById(R.id.rbFiveMinute);
+        rbOnline = (AppCompatRadioButton) view.findViewById(R.id.rbTenMinute);
+        rbOffline = (AppCompatRadioButton) view.findViewById(R.id.FifteenMinute);
```

Comment:
Why are these assigned to rbNormal, rbOnline and rbOffline?

Reply:
Well spotted.  Looks like when I refactored the names of the radio buttons in the new activity_bird_count.xml (which I must have copied from this code), it renamed them here also :-(
Now put back to correct names.